test/Game/GameToken.test.ts

```diff
@@ -133,8 +133,10 @@ describe('GameToken', function () {
       gameTokenAsMinter = await gameToken.connect(
         ethers.provider.getSigner(gameTokenAdmin)
       );
+      // For tests, minter should be set to deployed minter contract address
+      // on hardhat network.
       expect(await gameToken.getMinter()).to.be.equal(
-        ethers.constants.AddressZero
+        ethers.utils.getAddress('0x2fc631e4B3018258759C52AF169200213e84ABab')
```

Comment:
@nfurfaro is this always a constant?

Reply:
Hmm, good question. I was thinking it was, but not 100% now.
It would only be an annoyance if not, but would rather change it.
The minter address is set to the deployed minter contract by a deploy script, and so it's constant on Hardhat network.
The addition of new deployments **_before_** the AssetMinter deployment might change the deployer account nonce, and therefore the deployed address of the minter on Hardhat.